alerts/cluster-monitoring-operator/PrometheusOperatorRejectedResources.md

```diff
@@ -0,0 +1,88 @@
+# PrometheusOperatorRejectedResources
+
+## Meaning
+
+The `PrometheusOperatorRejectedResources` alert triggers when Prometheus Operator
+rejects invalid `Alertmanager`, `PodMonitor`, `ServiceMonitor`, or `PrometheusRules`
+objects.
+
+## Impact
+
+The custom resources that trigger the alert are ignored by Prometheus Operator.
+As a consequence, they will not be part of the final configuration of the
+Prometheus, Alertmanager, or Thanos Ruler components managed by Prometheus Operator.
+
+## Diagnosis
+
+Find the type of custom resource and its namespace in the `description` field
+of the `PrometheusOperatorRejectedResources` alert. You can find this information
+by using the OpenShift web console or the command line interface (CLI).
+
+### Diagnose using the Openshift web console
+
+1. Browse to **Observe** -> **Alerting**.
+2. Click the **Filter** drop-down list and select **Firing**.
```

Comment:
the Alerting page shows the Platform Firing alert by default, step 2 maybe not needed
![alerting](https://github.com/juzhao/test/assets/22743651/26abcaae-bf98-4d86-a6db-d54719ebdd54)

Reply:
I will change it to an advise.